fdbclient/BackupAgent.actor.h

```diff
@@ -679,6 +742,14 @@ class BackupConfig : public KeyBackedConfig {
 		return configSpace.pack(LiteralStringRef(__FUNCTION__));
 	}
 
+	KeyBackedProperty<int64_t> snapshotDispatchLastShardsBehind() {
+		return configSpace.pack(LiteralStringRef(__FUNCTION__));
+	}
+
+	KeyBackedProperty<Version> snapshotDispatchLastVersion() {
+		return configSpace.pack(LiteralStringRef(__FUNCTION__));
+	}
+
```

Comment:
These two new properties are not being read in this PR. I see that they are cleared in one place but i could not spot any reads of these properties. Am i missing something?

Reply:
Yes, it is read in FileBackupAgent.actor.cpp, which is where the JSON status is generated.  You probably just need to click "load diff" on that file, it's large enough that GitHub hides it by default.